src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpPrincipal.java

```diff
@@ -48,10 +50,13 @@ public HttpPrincipal (String username, String realm) {
     }
 
     /**
-     * Compares two HttpPrincipal. Returns <code>true</code>
-     * if <i>another</i> is an instance of HttpPrincipal, and its
-     * username and realm are equal to this object's username
-     * and realm. Returns <code>false</code> otherwise.
+     * Compare two instances of {@code HttpPrincipal}. Returns {@code true} if
+     * <i>another</i> is an instance of {@code HttpPrincipal}, and its username
+     * and realm are equal to this object's username and realm. Returns {@code false}
+     * otherwise.
+     *
```

Comment:
Isn't this missing an @param for `another` ?

Reply:
Well spotted. @param added in commit d8b93d01021d696f9f5272b1e6b34750dc36bcbf